### ISSUE TITLE
Add pass height information to SPADL

### ISSUE THREAD

**Issue by znstrider** [ML-KULeuven/socceraction#411]
Currently there is no information about pass height in socceraction.

I haven't checked it but I assume that the value of high passes could be significantly different from ground passes.

Do all providers offer this information and is that something that could be added to socceraction?

**Comment by probberechts** [comment]
I think all providers support it to some extent:

- Statsbomb distinguishes between ground passes, low passes (below shoulder level) and high passes.
- Wyscout distinguishes between low ( < 1m) and high passes, but this is only defined for crosses, free kicks and long passes
- Older Opta data has a qualifier for chipped passes. I do not have docs for more recent data.

One issue is that low / high has different meanings between providers.

I am considering adding this. Although, it has low priority. To some extent, the models should already be able to derive it from the body part used to perform the (previous) action(s) and the length of a pass.

**Comment by znstrider** [comment]
Agreed that the different definitions are a little problematic and there is definitely some of the info that is implicitly picked up by any model.
But in a way it will average over the expectation over both high and ground passes, which are different in a football sense and might differ in action value.

If I get around to testing a model with this information added, I'll let you know if it makes a meaningful difference.

I wonder if it makes sense to explicitly look at the difference of loss evaluations of high (& low) passes between models with/without that info additionally.

**Comment by probberechts** [comment]
> If I get around to testing a model with this information added, I'll let you know if it makes a meaningful difference.

That would be interesting!

> I wonder if it makes sense to explicitly look at the difference of loss evaluations of high (& low) passes between models with/without that info additionally.

I would look at the Brier / AUROC scores on an independent test set + evaluate a few examples where the models give different results qualitatively.
